recipes/mender-client-version-inventory-script/debian-master/control

```diff
@@ -0,0 +1,15 @@
+Source: mender-client-version-inventory-script
+Maintainer: Mender Team <mender@northern.tech>
+Section: devel
+Priority: optional
+Standards-Version: 1.0
+Build-Depends: debhelper (>= 9)
+Conflicts: mender (<< ${mender:Version}), mender (>> ${mender:Version})
```

Comment:
This I wrote in the JIRA task as an example to illustrate how it could be implemented. But of course this does not work. As I shared with you what we want here is the equivalent of doing `CONFLICTS_SEP="," make generate-conflicts` in the repo.

Note that `mender` package does not exist for example.